CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ and this project adheres to
 - 🐛(backend) fix s3 version_id validation
 - 🐛(frontend) retry check media status after page reload #1555
 - 🐛(frontend) fix Interlinking memory leak #1560
+- 🐛(frontend) preserve @ character when esc is pressed after typing it #1512
```

Comment:
We did a release, when you will rebase with `main`, think of moving this part under the `Unrelease` part.